### DISCUSSION TITLE
go-libp2p User Centric Community Call

### DISCUSSION THREAD

**Discussion by p-shahi** [libp2p/go-libp2p#3004] [General]
Hey go-libp2p community,
@MarcoPolo @sukunrt and I want to kick off a monthly user centric community call, where we'd like to:
*  present feature work and upkeep items we are working on (libp2p+HTTP, WebRTC, protocol hardening/security, etc.)
* discuss your priorities and needs from go-libp2p (both short term and long term)
* get feedback on our go-libp2p roadmap
* triage any issues you are running into with go-libp2p

Most importantly, we want to get to know you and create a place where different users from different can mingle and learn from one another!

We'd love to see you join us, and are thinking of kicking this off on **October 17th at 10 AM PST** : https://lu.ma/2024-10-17-go-libp2p

 cc: @Wondertan @algorandskiy @nisdas @nalepae @davidhadas @burdiyan @vyzo @marten-seemann @aschmahmann @yiannisbot

**Comment by p-shahi** [comment]
Hey all,

Just a reminder that this call is happening on **October 17th at 10 AM PST**: https://lu.ma/2024-10-17-go-libp2p

Thanks to those who have registered, and for those interested in joining, please sign up via the Luma link.
We're looking forward to getting to know our users and discuss go-libp2p with you all

Thanks!

**Comment by burdiyan** [comment]
I have some personal commitments this day, so I'm pretty sure I won't be able to make it. Maybe I'll try to join while on the go though.
